src/frontend/components/ProfileCard/ProfileCard.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { User } from '../User/User'
+import ProfileCardCSS from "./ProfileCard.module.css"
+import linkedin from "../../assets/linkedin.png"
+import github from "../../assets/github.png"
+import twitter from "../../assets/twitter.png"
+// import { PostCard } from '../PostCard/PostCard'
+
+function ProfileCard() {
+  return (
+    <div className={ProfileCardCSS.container}>
+        <div className={`d-flex ${ProfileCardCSS.header}`}>
+            <User/>
+            <span>Edit</span>
+        </div>
+        <article className={ProfileCardCSS.description}>
+            Aspiring Front-End Developer
+        </article>
+        <ul className={`d-flex ${ProfileCardCSS.stats}`}>
+            <li>3 Posts</li>
+            <li>5 Followers</li>
+            <li>2 Following</li>
+        </ul>
+        <a href="https://www.linkedin.com/in/hanand2021" target={"_blank"} className={ProfileCardCSS.social_media}>
```

Comment:
No need to add curly braces around `_blank`

Reply:
Okay. Will make the changes as suggested